libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
In ARMCC,
http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0491bj/CJAFBCBB.html.
In GCC ARM,
\libraries\mbed\targets\cmsis\core_caFunc.h line840-846.